cdap-ui/app/directives/dag/my-dag.less

```diff
@@ -41,6 +41,13 @@ my-dag {
       background-image: none;
       border: 0;
       background-color: @table-bg;
+
+      &.normal-cursor {
```

Comment:
We anyways have a ```#dag-container``` in the html. Is it possible to have for everything under it to be cursor default? Something like,

```
my-dag {
  &.disabled {
    &.normal-cursor {
      #dag-container {
        * { cursor: default; }
       }
    } 
  }
}
```